### PR TITLE
Sidebar background color fixed

### PR DESCRIPTION
**Pull request by aliashkevich**
Will create ticket to refactor header to use React (in small screen size after clicking NavLink sidebar is not closed).

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -105,7 +110,7 @@ export default class Header extends Component {
                   <li className='nav-item dropdown'>
                     <a
                       className='nav-link'
-                      href='#pablo'
+                      href='javascript:void(0)'
```
Comment (jenna1k) [inline src/components/Header.js:113]: what does it do?

File: src/App.js [hunk 2]
```diff
@@ -13,24 +11,38 @@ import NewProject from './pages/NewProject';
 import NewClient from './pages/NewClient';
 import Tasks from './pages/Tasks';
 import Users from './pages/Users';
+import Header from './components/Header';
 
 function App() {
+  const LoginContainer = () => (
```
Comment (jenna1k) [inline src/App.js:17]: thank you for dividing container so we don't have to put `<Header/>` everywhere :heart_eyes:

**Review by jenna1k — APPROVED** [review]

File: src/components/Header.js (hunk 1 above)
Comment (aliashkevich) [reply src/components/Header.js:113]: https://stackoverflow.com/questions/22940761/best-way-to-create-an-a-link-with-empty-href 
Just decided to replace link to not existing pablo page, but this link should lead nowhere) and it's a way how to use empty href

File: src/App.js (hunk 2 above)
Comment (aliashkevich) [reply src/App.js:17]: ^_^

**Review by wilsoncooked — APPROVED** [review]

**Pull request merged**